src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsExpandable.js

```diff
@@ -0,0 +1,49 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import './CVEPageDetailsDescription.scss';
+import { DESCRIPTION_CHARACTER_LIMIT } from '../../../Helpers/constants';
```

Comment:
you can make the component even smarter if you allow that param to be as `prop`, this means that whenever you call the component you can define how long or short the text would be